config/routes.rb

```diff
@@ -1,8 +1,16 @@
 Rails.application.routes.draw do
 
   resources :companies do
-    resources :jobs
+    resources :contacts
+    resources :jobs do
+      resources :comments, only: [:create]
```

Comment:
we talked about this last week - and nesting three deep. let me know if you have questions about this 👍